lib/jnpr/junos/rpcmeta.py

```diff
@@ -25,9 +25,10 @@ def get_config(self, filter_xml=None, options={}):
         """
         retrieve configuration from the Junos device
 
-        :filter_xml: is options, defines what to retrieve.  if omitted then the entire configuration is returned
+        :filter_xml: XPath expression consisting of XML tag names separated by slashes,
+                     defines what to retrieve; when omitted then the entire configuration is returned
 
-        :options: is a dict, creates attributes for the RPC
+        :options: is a dictionary of XML attributes to set within the <get-configuration> RPC.
```

Comment:
@astiphout Can you please also add example of :options: in the next line for reference. Even better if we can have example for filter_xml